addons/pos_loyalty/__openerp__.py

```diff
@@ -20,13 +20,13 @@
     'author': 'OpenERP SA',
```

Comment:
According to https://github.com/odoo/odoo/commit/0decfd94d13fb49fe9a1e92f8aacd83a0cddf0b7 commit I remove author.